Add_Keyboard_Support/DriverImplementation.md

```diff
@@ -0,0 +1,217 @@
+# Handling scancodes
+
+Handling the keyboard interrupt was pretty simple, we just basically needed to add an irq to the APIC and read the scancode. But at the end what we got is just a number, and we want to eventually translated it into a character. 
+
+In this section we will see see how to build the Keyboard driver, and what are its responsibilities. 
+
+First of all the driver is not responsible of translating the scancode into printable characters (well sometime on early stages of development it does it too...)
```

Comment:
yes! Although I would say its responsible for translating keyboard specific scancodes (set 1/2/3) into some vendor neutral format that any program can use.